analyses/clone-phylogeny-analysis/01-create-count-mat.R

```diff
@@ -53,7 +54,7 @@ if (!dir.exists(step_results_dir)) {
 
 #######################################################
 # Read metadata file and define `sample_name`
-metadata_file <- file.path(metadata_dir, "project_metadata.tsv") # metadata input file
+metadata_file <- file.path(metadata_dir, metadata_file) # metadata input file
```

Comment:
Same commenting regarding the `metadata_file` variable here and on line `60`.

```suggestion
project_metadata_file <- file.path(metadata_dir, metadata_file) # metadata input file
```